components/card/index.css

```diff
@@ -60,196 +188,282 @@ governing permissions and limitations under the License.
   }
 }
 
-.spectrum-Card-actions {
+.spectrum-Card .spectrum-QuickActions {
```

Comment:
Leveraging our new custom properties over nested class names approach, are there mods we can hook into on the Quick actions component rather than including the class here?

Reply:
Good point - I hadn't even thought about that factor!